src/HotChocolate/Fusion-vnext/src/Fusion.Execution.Types/Completion/CompletionTools.cs

```diff
@@ -77,6 +77,25 @@ public static FusionInterfaceTypeDefinitionCollection CreateInterfaceTypeCollect
         return new FusionInterfaceTypeDefinitionCollection(temp);
     }
 
+    public static FusionObjectTypeDefinitionCollection CreateObjectTypeCollection(
+        IReadOnlyList<NamedTypeNode> types,
+        CompositeSchemaContext context)
+    {
+        if (types.Count == 0)
```

Comment:
I think we can ignore this case as unions without any object types are invalid. This is different for interfaces ... as an object is allowed to have no interfaces.